api-tests/src/types/world.ts

```diff
@@ -26,7 +26,7 @@ export interface World extends CucumberWorld {
   mfaResetResult?: MfaResetResult;
 
   // Healthcheck results
-  healthCheckResult?: boolean;
+  healthCheckResult?: string;
```

Comment:
It's not really needed, but means that we're checking the `healthcheck` property in the response rather than the status code :shrug:

Would also be happy to revert and just `return true`